src/test/java/com/unzer/payment/service/UrlUtilTest.java

```diff
@@ -21,54 +21,81 @@
  */
 
 
-import com.unzer.payment.Unzer;
 import com.unzer.payment.business.AbstractPaymentTest;
+import com.unzer.payment.business.Keys;
 import com.unzer.payment.business.paymenttypes.InstallmentSecuredRatePlan;
 import com.unzer.payment.communication.HttpCommunicationException;
 import com.unzer.payment.paymenttypes.InvoiceSecured;
+import com.unzer.payment.paymenttypes.PaymentType;
 import com.unzer.payment.paymenttypes.SepaDirectDebitSecured;
+import org.junit.Rule;
 import org.junit.Test;
+import uk.org.webcompere.systemstubs.rules.EnvironmentVariablesRule;
 
 import static org.junit.Assert.assertEquals;
 
 public class UrlUtilTest extends AbstractPaymentTest {
+    @Rule
+    public EnvironmentVariablesRule environmentVariablesRule = new EnvironmentVariablesRule();
+
+    private static final String sbxTypesUrl = "https://sbx-api.unzer.com/v1/types/";
+    private static final String id = "random-id";
+
+    @Test
+    public void whenProductionKey_returnsProductionEndpoint() {
+        runDifferentEnvironmentTest("p-random-key", "", "https://api.unzer.com");
+    }
```

Comment:
To be sure that production key always use production url, I would do the same test with "dev" and "stg" env set.